### DISCUSSION TITLE
报错An unexpected error occurred

### DISCUSSION THREAD

**Discussion by rumosky** [walinejs/waline#1402] [Q&A] [answered]
如图，报错，一直是这样，重试了很多遍
![QQ截图20220914205219](https://user-images.githubusercontent.com/41993388/190159154-25f3ab2d-1b5f-4503-aa39-e323526b4ec7.png)

**Comment by lizheming** [comment, marked as answer]
建议提 Vercel 工单

**Reply by rumosky** [reply]
ok
